src/renderer/services/keystore/eth2.ts

```diff
@@ -58,18 +49,17 @@ export class Eth1Keystore implements ICGKeystore {
      */
     public async changePassword(oldPassword: string, newPassword: string): Promise<void> {
         const keypair = await this.decrypt(oldPassword);
-        const keystore = await Eth1Keystore.createKeystoreObject(newPassword, keypair);
-
+        const keystore = Keystore.encrypt(keypair.privateKey.toBytes(), newPassword);
         try {
-            writeFileSync(this.file, JSON.stringify(keystore, null, 2));
+            writeFileSync(this.file, keystore.toJSON());
             this.keystore = keystore;
         } catch (err) {
             throw new Error(`Failed to write to ${this.file}: ${err}`);
         }
     }
 
     public getAddress(): string {
-        return this.keystore.address;
+        return this.keystore.pubkey.toString();
```

Comment:
I would maybe expose another method getPublicKey and make this method convert public key to address using bech32